pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1563,6 +1571,8 @@ public void checkCompaction() {
                 }
 
                 if (backlogEstimate > compactionThreshold) {
+                    log.info("topic:{} backlogEstimate:{} is bigger than compactionThreshold:{}. Triggering compaction",
```

Comment:
It should be a debug-level log.

Reply:
I thought this topic compaction log is useful. Could you share your concerns about this?

Reply:
Suppose you have many topics, like 50k topics per broker. By default, we will have 50k logs per minute. And we already have logs after the compaction task started.

Reply:
Ok. I will make this debug lvl.

Reply:
Updated.